### ISSUE TITLE
MS 0073 very slow loading images

### ISSUE THREAD

**Issue by napulen** [DDMAL/cantus#558]
From all the manuscripts that are currently online in staging, it seems that `MS 0073` is much slower at fetching the images.

It'd be nice to know why this is happening, is it a bandwidth problem, or is there something about the IIIF/URLs/etc. in this manuscript that is not working properly.

**Comment by annamorphism** [comment]
@napulen For what it's worth there are other McGill manuscripts I've tried to load into Fragmentarium via the manifest and they are also really notably slow to load. Then once they've been loaded and presumably cached they work just fine. 

I've assumed it was an image size/bandwidth problem (the TIFF files are hundreds of MB), but maybe it's the manifest itself. Whatever it is seems to be something consistent across McGill's images.

**Comment by fujinaga** [comment]
I think McGill images are hosted on Internet Archives.

**Comment by annamorphism** [comment]
@fujinaga they are; but I've noticed it takes a pretty long time for them to load in other IIIF viewers (Mirador vel sim) compared to other images on Internet Archive (from other institutions). McGill's images are also noticeably higher-res than a lot of other things on Internet Archive (or e.g. Gallica for that matter).  Since the load issues I've experienced seem to relate to McGill's items in general but not IA's items in general, my guess would be that the issue has to do with the required bandwidth rather than the manifest/URL (which I would think would behave the same way across Internet Archive).

**Comment by fujinaga** [comment]
Yes, it would be a combination of the bandwidth that IA provides and that McGill images are relatively large. We just have to wait until either IA increases its bandwidth or McGill decides to host its images elsewhere with faster bandwidth. One thing we may consider is to copy McGill's music-related images and host them ourselves on Compute Canada, at least for the next 7 years or so.
